### PR TITLE
Feature/sc 25513/integrate organization dropdown component

### PR DESCRIPTION
**Pull request by bereketsega**
This PR adds organizations dropdown options component on `edit-profile` component.

## Demo

https://github.com/Cyber4All/clark-client/assets/72763770/a2c4c8c0-f8c7-40f5-885d-2d3613d26c4a

### CODE REVIEW

**Review by stef359 — APPROVED** [review]

File: src/app/core/interceptor/httpconfig.interceptor.ts [hunk 1]
```diff
@@ -20,7 +21,9 @@ export class HttpConfigInterceptor implements HttpInterceptor {
     next: HttpHandler
   ): Observable<HttpEvent<any>> {
     let headers = request.headers.set('Content-Type', 'application/json');
+    if (!request.url.includes(environment.cardOrganizationUrl)) {
```
Comment (mfranz1) [inline src/app/core/interceptor/httpconfig.interceptor.ts:24]: Nice

File: src/app/cube/user-profile/components/edit-profile/edit-profile.component.html [hunk 2]
```diff
@@ -41,7 +41,17 @@ <h1>EDIT YOUR PROFILE</h1>
         #organization 
         phold="Organization" 
         [control]="editFormGroup.get('organization')"
+		(keyup)="keyup($event)"
```
Comment (mfranz1) [inline src/app/cube/user-profile/components/edit-profile/edit-profile.component.html:44]: nit pick can you tab this over

**Review by mfranz1 — APPROVED** [review]
This looks good

**Review by shanioffe — APPROVED** [review]

**Review by JosueMolinaMorales — APPROVED** [review]

**Review by katieblomquist — CHANGES_REQUESTED** [review]
Overall it looks pretty good! My only suggestion would be to make sure your dropdown is laying over the components below it rather than pushing them out of the way.

**Comment by bereketsega** [comment]
> Overall it looks pretty good! My only suggestion would be to make sure your dropdown is laying over the components below it rather than pushing them out of the way.

**UPDATED**
Screenshot 2023-09-12 at 1 36 12 PM

**Review by katieblomquist — APPROVED** [review]

**Pull request merged**